gno2go.go

```diff
@@ -0,0 +1,129 @@
+package gno
+
+import (
+	"bytes"
+	"fmt"
+	"go/ast"
+	"go/format"
+	"go/token"
+	"strings"
+
+	"go.uber.org/multierr"
+	"golang.org/x/tools/go/ast/astutil"
+)
+
+const (
+	gnoRealmPkgsPrefixBefore = "gno.land/r/"
+	gnoRealmPkgsPrefixAfter  = "github.com/gnolang/gno/examples/gno.land/r/"
+	gnoPackagePrefixBefore   = "gno.land/p/"
+	gnoPackagePrefixAfter    = "github.com/gnolang/gno/examples/gno.land/p/"
+	gnoStdPkgBefore          = "std"
+	gnoStdPkgAfter           = "github.com/gnolang/gno/stdlibs/stdshim"
+)
+
+var stdlibWhitelist = []string{
+	"regexp",
+	"std",
+	"strconv",
+	"strings",
+	"maths",
+}
+
+func Gno2Go(fset *token.FileSet, f *ast.File) (string, error) {
```

Comment:
To disambiguate from Go2Gno which works on values (and there may be a need for Gno2Go for values too), maybe this should be renamed to Gno2GoPrecompile.

Reply:
fixed